common/numpy_fast.py

```diff
@@ -1,2 +0,0 @@
-def clip(x, lo, hi):
```

Comment:
I did not know that, but even so I question why it gets its own module in common when it was only used a few times.